views/index.erb

```diff
@@ -0,0 +1,23 @@
+<div class="new_post">
+  <form method="post" action="/new">
+    <textarea class="text-input" name="message" placeholder="Enter a peep (max 200 char)..."></textarea><br>
+    <input class="submit button" type="submit" value="Post">
+  </form>
+
+  <%= flash[:empty] %>
```

Comment:
So I'm guessing this flash doesn't come up if the empty key of the flash param has not been populated.